.buildkite/scripts/benchmark/config/filebeat.yml

```diff
@@ -0,0 +1,17 @@
+http.enabled: false
+filebeat.inputs:
+- type: log
+  symlinks: true
+  paths:
+  - "/usr/share/filebeat/flog/*.log"
+logging.level: info
+output.logstash:
+  hosts:
+    - "localhost:5044"
+  compression_level: 0
+  pipelining: 3
```

Comment:
interesting, why 3? I'd suggest leaving the default here.

Reply:
Agree to maintain the default behavior. 
Base on my last benchmark test, 3 gave more eps.